specialcharacters.go

```diff
@@ -0,0 +1,29 @@
+package goaway
+
+var DefaultSpecialCharacters = []rune("._-?()|~")
+
+func createReplacementMap(runes []rune) map[rune]rune {
+	specialReplacementMap := make(map[rune]rune, len(runes))
+	for _, character := range runes {
+		specialReplacementMap[character] = ' '
+	}
+	return specialReplacementMap
+}
+
+// DefaultLeetspeekCharactersReplacement contains list of special character runes that will be turned to special character mapping
+var DefaultSpecialCharacterReplacements = createReplacementMap(DefaultSpecialCharacters)
```

Comment:
Does it really make sense to name them replacements, given that they're technically skipped/ignored/trimmed?

Reply:
good catch. I renamed and removed unnecessary variable